frontend/src/app/features/work-packages/directives/query-space/isolated-query-space.ts

```diff
@@ -80,6 +85,6 @@ export class IsolatedQuerySpace extends StatesGroup {
     filters: input<QueryFilterInstanceSchemaResource[]>(),
 
     // Display of the WP results
-    displayRepresentation: input<WorkPackageDisplayRepresentationValue|null>(),
+    displayRepresentation: input<WorkPackageDisplayRepresentationValue | null>(),
```

Comment:
Somewhat useless edit, probably from your IDE?

Reply:
Yeah, that was autofixed on save, will undo